src/Core/Managed/Shared/Extensibility/Metrics/Metric.cs

```diff
@@ -1,13 +1,12 @@
-﻿namespace Microsoft.ApplicationInsights.Extensibility
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Metrics
```

Comment:
This is new functionality. was planning to update the Web SDK to adjust to this change, but not other tooling, which should be updated next time it is touched.
There are three reasons for this change:
a) internal. since this is a current area of investment, separating metrics specific code helps organizing.
b) user facing. since there are a whole bunch of public APIs that only make sense in the context of metrics, this will help keep intellisense clutter-free for customers who do not need these APIs and do not import ...Metrics.
c) Some nuances form this discussion: https://github.com/Microsoft/ApplicationInsights-dotnet/issues/439